src/components/application_manager/src/commands/command_impl.cc

```diff
@@ -214,5 +214,24 @@ DEPRECATED void CommandImpl::ReplaceHMIByMobileAppId(
   }
 }
 
+bool CommandImpl::CheckSyntax(const std::string& str,
```

Comment:
@ZhdanovP could be const

Reply:
@AByzhynar @ZhdanovP  CommandImpl::CheckSyntax should not change internals of command so it should be conts